jupyter-docker/Dockerfile

```diff
@@ -1,34 +1,30 @@
 # adapted from https://hub.docker.com/r/jupyter/base-notebook/ AKA https://github.com/jupyter/docker-stacks/tree/master/base-notebook
 
-# Debian Jessie debootstrap from 2017-02-27
-# https://github.com/docker-library/official-images/commit/aa5973d0c918c70c035ec0746b8acaec3a4d7777
-FROM debian@sha256:52af198afd8c264f1035206ca66a5c48e602afb32dc912ebf9e9478134601ec4
+FROM debian:jessie
```

Comment:
Is ok to use the `debian:jessie` tag instead of a specific hash? It would keep us up to date on patches, etc. Other images use `FROM openjdk:8` for example.

Reply:
We'll be pinning this soon anyway with the "pin versions" story.